modules/SofaGeneralLoader/GeneralLoader.cpp

```diff
@@ -1,35 +0,0 @@
-/******************************************************************************
```

Comment:
Why did you removed also code ?

Reply:
yes, I saw all includes could be removed because this class is not anymore in the cmakeLists.
Another PR regarding deprecated files will be better indeed because I found other similar cases.